docs/user-guide/sync_windows.md

```diff
@@ -2,7 +2,31 @@
 
 Sync windows are configurable windows of time where syncs will either be blocked or allowed. These are defined
 by a kind, which can be either `allow` or `deny`, a `schedule` in cron format and a duration along with one or 
-more of either `applications`, `namespaces` and `clusters`. Wildcards are supported. These windows affect the running 
+more of either `applications`, `namespaces` and `clusters`. Wildcards are supported. 
+
+## Relationship between Sync Windows and Applications
+
+The relationship between Sync Windows and Application resources is many-to-many. This means that an Application resource
+may be affected by multiple Sync Windows, and that a single Sync Window definition may apply to multiple Application
+resources. 
+
+The relationship between Sync Window and Application is established as part of the definition of Sync Window.
+Sync Window definition includes a section defining the Application resources to which it applies. There
+are three mechanisms for selecting the Application resources to which a Sync Window applies:
+
+- By name of Application resource
+- By cluster into which resources are installed by Application resource. This is specified by `Application.spec.destination.name` and `.server` fields
+- By namespace into which resources are installed by Application resource. This is specified by `Application.spec.destination.namespace` field.
+
+All three mechanisms allow usage of wildcards. The mechanisms are not mutually exclusive, and all three of them can be used in single
+Sync Window definition. 
+
+When multiple selection mechanisms are used, they are effectively `ORed`, meaning that if any of the selector selects the Application,
```

Comment:
I've raised a while ago a PR for adding also `AND` support. https://github.com/argoproj/argo-cd/pull/16846 is still open, not sure if you want to mentioned it here or I can update the docs once that PR is close to being merged.

Reply:
Hi @adriananeci, I think that proper way forward here is to have your branch rebased onto main after the documentation here is merged, and then to update the documentation in your branch. 

If we change the documentation now, we will be adding documentation for features, which are not yet included in the mainline, which would be incorrect.

Reply:
I agree with @vaniobegic here. @adriananeci , you would want to add/update the documentation as part of https://github.com/argoproj/argo-cd/pull/16846